bots/pr/src/main/java/org/openjdk/skara/bots/pr/IntegrateCommand.java

```diff
@@ -108,6 +121,9 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
                 if (!ProjectPermissions.mayCommit(censusInstance, pr.author())) {
                     reply.println(ReadyForSponsorTracker.addIntegrationMarker(pr.headHash()));
                     reply.println("Your change (at version " + pr.headHash() + ") is now ready to be sponsored by a Committer.");
+                    if (!args.isBlank()) {
+                        reply.println("Note that your sponsor will make the final decision on which target hash to integrate on.");
```

Comment:
```suggestion
                        reply.println("Note that your sponsor will make the final decision on which target hash to integrate upon.");
```

Reply:
How about onto, as in `git rebase --onto`?

Reply:
:+1: